### ISSUE TITLE
when I added into tsconfig.json some outDir, the test with use your library has crashed

### ISSUE THREAD

**Issue by nskazki** [power-assert-js/espower-typescript#14]
hello, sorry for my english.
when I added into **tsconfig.json** some **outDir**, the test with use your library has crashed

small research: 
- mocha load test file: http://i.imgur.com/w2QHaDf.png
- module call espower-typescript: http://i.imgur.com/z1wcEgo.png
- espower-typescript call simple-typescript, **but lose filename**: http://i.imgur.com/9fiUsnM.png
- some internall function make incorrect resolve: http://i.imgur.com/tnOaDOX.png
- and eventually turns a broken file name: http://i.imgur.com/hSYeFRc.png
- so, computed name http://i.imgur.com/TCwm24d.png dosn't match the default name http://i.imgur.com/tyF1WLA.png
- file = undefined http://i.imgur.com/mtFBOcd.png and process crashed http://i.imgur.com/sbXdNvw.png

I think that is enough pass the lost name of the file, to solve the problem.

**Comment by nskazki** [comment]
example - https://github.com/nskazki/espower-typescript-issue-example

**Comment by nskazki** [comment]
I'm not going to do a PR, because this error not reproduced in your repository (https://github.com/power-assert-js/espower-typescript), and the project is too large to be sure that I do everything correctly. but it works for me:
```
  function loadTypeScript(localModule, filepath) {
    var result = tss.compile(fs.readFileSync(filepath, 'utf-8'), filepath.replace(process.cwd(), ''));
    // var result = tss.compile(fs.readFileSync(filepath, 'utf-8'));
    if (minimatch(filepath, pattern)) {
      result = espowerSource(result, filepath, options);
    }
    localModule._compile(result, filepath);
  };
```

**Comment by teppeis** [comment]
@nskazki Thanks for nice report! I've confirmed this issue.

**Comment by teppeis** [comment]
@nskazki fixed in `v5.0.1`

**Comment by nskazki** [comment]
@teppeis Many thanks, I checked on my example - now works with a **outDir** parameter.
